common/v2/features/Dashboard/Dashboard.tsx

```diff
@@ -98,30 +98,6 @@ export default function Dashboard() {
                 </div>
               </div>
             </div>
-            <AddressMetadataContext.Consumer>
-              {({ readAddressMetadata }) => (
-                <TransactionContext.Consumer>
-                  {({ transactions }) => {
-                    return (
-                      <TransactionHistoryContext.Consumer>
-                        {({ transactionHistories }) => {
-                          return (
-                            <div className="Dashboard-desktop-bottom">
-                              <RecentTransactionList
-                                transactionHistories={transactionHistories}
-                                readAddressMetadata={readAddressMetadata}
-                                transactions={transactions}
-                                className="Dashboard-desktop-modifiedPanel"
-                              />
-                            </div>
-                          );
-                        }}
-                      </TransactionHistoryContext.Consumer>
-                    );
-                  }}
-                </TransactionContext.Consumer>
-              )}
-            </AddressMetadataContext.Consumer>
```

Comment:
Do we still need to keep these contexts around for other parts of the GAU MVP code?

Reply:
probably not especially with the data structure rework